crates/primitives/src/log.rs

```diff
@@ -54,8 +20,43 @@ where
 
 #[cfg(test)]
 mod tests {
-    use super::*;
+    use alloy_primitives::{Address, Bytes, Log as AlloyLog, B256};
+    use alloy_rlp::{RlpDecodable, RlpEncodable};
     use proptest::proptest;
+    use reth_codecs::{main_codec, Compact};
+
+    #[main_codec(rlp)]
```

Comment:
can add a comment here why we have this,

ref: https://github.com/paradigmxyz/reth/pull/7636